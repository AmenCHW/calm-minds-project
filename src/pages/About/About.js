import React from 'react';

function About() {
  return <h1 className="mx-auto text-center text-7xl">About</h1>;
}

export default About;
