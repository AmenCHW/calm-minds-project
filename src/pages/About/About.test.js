
import React from "react";
import {render,screen} from "@testing-library/react"
import About from "./About";
import renderer from 'react-test-renderer' 

// _________________getAllByText_________________
test("test healing", async() => {
    render(
       <About title ="Healing" />
   );
 const headingElement = screen.getAllByText(/healing/i);
   expect(headingElement).toBeInTheDocument;
 });


// _________________testing title with getByText_________________
it("test <p> ", async() => {
     render(
        <About title="about" />
  );
    const headingElement = screen.getByText(/About/i);  //case sensittive?
    expect(headingElement).toBeInTheDocument();
   });


   // last one ,snapshot
 test("first snapshot testing",()=>{
    const component=renderer.create( <About/>).toJSON()  
    expect( component).toMatchSnapshot();
})


test('first snapshot testing', () => {
  const component = renderer.create(<About />).toJSON();
  expect(component).toMatchSnapshot();
});