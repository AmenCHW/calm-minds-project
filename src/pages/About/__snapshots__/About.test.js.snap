// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot testing 1`] = `
<div
  className=""
>
  <div
    className="mx-auto text-center px-10 md:px-32 lg:px-48 sm:text-left bg-white font-poppins pt-12"
  >
    <h2
      className="font-normal text-5xl pb-2"
    >
      HEALING!
    </h2>
    <h3
      className="font-light text-2xl text-[#808080] pb-16 leading-6"
    >
      some cool one liner !
    </h3>
    <p
      className="font-normal leading-7 text-[#424A4F] text-xl pb-20 sm:pb-40"
    >
      At Healing, we believe there is a better way to do things. A more valuable way where customers are earned rather than bought. We're obsessively passionate about it, and our mission is to help people achieve it. We focus on search engine optimization. It's one of the least understood and least transparent aspects of great marketing, and we see that as an opportunity. We're excited to simplify SEO for everyone through our software, education, and community.
    </p>
  </div>
  <div
    className="mx-auto text-center px-10 md:px-32 lg:px-48 sm:text-left bg-[#EAF8F9] font-poppins pt-10 flex flex-col place-items-center sm:flex sm:flex-row sm:place-items-start"
  >
    <div
      className="w-2/5  sm:pr-14 pb-10"
    >
      <img
        alt="Founder Pic"
        className="rounded-md"
        src="therapist.jpg"
      />
    </div>
    <div
      className="w-3/5 pb-10"
    >
      <h2
        className="text-[#424A4F] font-normal text-5xl pb-7"
      >
        Our Founding
      </h2>
      <p
        className="text-xl"
      >
        Healing was founded by Payam Abubakr in 2021. It was called Healing Online, and started as a blog and an online community where some of the world's therapists shared their research and ideas. We launched the Beginner's Guide to Therapy and our first study, and that hub of industry expertise transformed into a small consulting firm and led us to create the Online Therapist of today!
      </p>
    </div>
  </div>
</div>
`;

exports[`first snapshot testing 2`] = `
<div
  className=""
>
  <div
    className="mx-auto text-center px-10 md:px-32 lg:px-48 sm:text-left bg-white font-poppins pt-12"
  >
    <h2
      className="font-normal text-5xl pb-2"
    >
      HEALING!
    </h2>
    <h3
      className="font-light text-2xl text-[#808080] pb-16 leading-6"
    >
      some cool one liner !
    </h3>
    <p
      className="font-normal leading-7 text-[#424A4F] text-xl pb-20 sm:pb-40"
    >
      At Healing, we believe there is a better way to do things. A more valuable way where customers are earned rather than bought. We're obsessively passionate about it, and our mission is to help people achieve it. We focus on search engine optimization. It's one of the least understood and least transparent aspects of great marketing, and we see that as an opportunity. We're excited to simplify SEO for everyone through our software, education, and community.
    </p>
  </div>
  <div
    className="mx-auto text-center px-10 md:px-32 lg:px-48 sm:text-left bg-[#EAF8F9] font-poppins pt-10 flex flex-col place-items-center sm:flex sm:flex-row sm:place-items-start"
  >
    <div
      className="w-2/5  sm:pr-14 pb-10"
    >
      <img
        alt="Founder Pic"
        className="rounded-md"
        src="therapist.jpg"
      />
    </div>
    <div
      className="w-3/5 pb-10"
    >
      <h2
        className="text-[#424A4F] font-normal text-5xl pb-7"
      >
        Our Founding
      </h2>
      <p
        className="text-xl"
      >
        Healing was founded by Payam Abubakr in 2021. It was called Healing Online, and started as a blog and an online community where some of the world's therapists shared their research and ideas. We launched the Beginner's Guide to Therapy and our first study, and that hub of industry expertise transformed into a small consulting firm and led us to create the Online Therapist of today!
      </p>
    </div>
  </div>
</div>
`;
